package.json

```diff
@@ -17,7 +17,7 @@
         "build:webpack": "webpack --mode=production",
         "prestart": "npm run build",
         "start": "node tools/dev-server.js",
-        "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
+        "build": "npm-run-all build:sass --parallel install:playground build:eleventy build:webpack images",
```

Comment:
Shouldn't `build:webpack` wait for` install:playground` to finish?

Reply:
Also, `images` script works directly on `_site/assets/images`, so it seems that it should wait for `build:eleventy` to copy the images there (i.e., to finish).

Reply:
Yes, I don’t think we can safely do these in parallel. Nice idea, just doesn’t work in this case.